enterprise/backend/test/metabase_enterprise/audit_app/pages/common_test.clj

```diff
@@ -20,12 +19,11 @@
 
 (defmethod audit.i/internal-query ::legacy-format-query-fn
   [_ a1]
-  (let [h2? (= (mdb/db-type) :h2)]
```

Comment:
`h2?` here was unused so removing it actually let us eliminate a namespace dependency